### DISCUSSION TITLE
Get list of jars which required for sbt-launcher

### DISCUSSION THREAD

**Discussion by ppllaxxa** [sbt/sbt#7136] [Q&A]
Hello, I am trying to get full list of jars which will be downloaded by launching of sbt-launcher.jar. I need to identify full list to be able to add it to our internal repo, we couldn't use any external repo by policy. I tried just setup our internal repo for project and run sbt and collect jar names which were failed to download but I am sure that it is not full list because of transient dependencies.
Please advise where I could find full list of jars (group/artefact/version)?

**Comment by eed3si9n** [comment]
It should be `org.scala-sbt:sbt:1.8.2` I think. In addition, Zinc probably requires compiler bridge source code - https://repo1.maven.org/maven2/org/scala-sbt/compiler-bridge_2.13/1.8.0/
